doc/classes/Label.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Label" inherits="Control" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Displays plain text in a line or wrapped inside a rectangle. For formatted text, use [RichTextLabel].
+		A control for displaying plain text.
 	</brief_description>
 	<description>
-		Label displays plain text on the screen. It gives you control over the horizontal and vertical alignment and can wrap the text inside the node's bounding rectangle. It doesn't support bold, italics, or other formatting. For that, use [RichTextLabel] instead.
-		[b]Note:[/b] Contrarily to most other [Control]s, Label's [member Control.mouse_filter] defaults to [constant Control.MOUSE_FILTER_IGNORE] (i.e. it doesn't react to mouse input events). This implies that a label won't display any configured [member Control.tooltip_text], unless you change its mouse filter.
+		A control for displaying plain text. It gives you control over the horizontal and vertical alignment and can wrap the text inside the node's bounding rectangle. It doesn't support bold, italics, or other rich text formatting. For that, use [RichTextLabel] instead.
+		[b]Note:[/b] Contrary to most other [Control]s, [Label]'s [member Control.mouse_filter] defaults to [constant Control.MOUSE_FILTER_IGNORE] (i.e. it doesn't react to mouse input events). This implies that a label won't display any configured [member Control.tooltip_text], unless you change its mouse filter.
```

Comment:
We have a configuration warning for that. I wonder if it's worth keeping this note 🤔

Reply:
I usually removed notes/warnings that have counterpart configuration warnings throughout my rewriting. I'll remove this note.

I consider that configuration warnings are still useful knowledge when reading the documentation; maybe in the future we should introduce some way to add configuration warnings to the documentation.

Reply:
But in this case the warning applies to all Controls, not only the Label. The note was only here, because Label happens to use a different default and some random person was confused and opened an issue.